content/fr/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -0,0 +1,355 @@
+à posséder les pods spécifiés par son modèle - il peut acquérir d’autres pods de la manière spécifiée dans les sections précédentes.
+
+Prenez l'exemple précédent de ReplicaSet, ainsi que les pods spécifiés dans le manifeste suivant :
+
+{{< codenew file="pods/pod-rs.yaml" >}}
+
+Ces pods n’ayant pas de contrôleur (ni d’objet) en tant que référence propriétaire, ils correspondent au sélecteur de du ReplicaSet frontend, ils seront donc immédiatement acquis par ce ReplicaSet.
+
+Suppose you create the Pods after the frontend ReplicaSet has been deployed and has set up its initial Pod replicas to
+fulfill its replica count requirement:
+
+Supposons que vous créiez les pods une fois le ReplicaSet frontend déployé et qui a déjà déployé ses replicas de Pods initiaux afin de
+remplir son exigence de nombre de replicas :
+
+```shell
+kubectl apply -f https://kubernetes.io/examples/pods/pod-rs.yaml
+```
+
+Les nouveaux pods seront acquis par le ReplicaSet, puis immédiatement terminés car le ReplicaSet dépasserait alors le compte désiré.
+
+En récupérant les pods :
+```shell
+kubectl get Pods
+```
+
+La sortie montre que les nouveaux pods sont soit déjà terminés, soit en voie de l'être :
+```shell
+NAME             READY   STATUS        RESTARTS   AGE
+frontend-9si5l   1/1     Running       0          1m
+frontend-dnjpy   1/1     Running       0          1m
+frontend-qhloh   1/1     Running       0          1m
+pod2             0/1     Terminating   0          4s
+```
+
+Cependant, si vous créez d'abord les pods :
+```shell
+kubectl apply -f https://kubernetes.io/examples/pods/pod-rs.yaml
+```
+
+Et puis créez le ReplicaSet :
+```shell
+kubectl apply -f https://kubernetes.io/examples/controllers/frontend.yaml
+```
+
+Vous verrez que le ReplicaSet a acquis les pods et n'a créé que les nouveaux Pods manquants, conformément à ses spécifications,
+jusqu'au nombre souhaité de Pods. En récupérant les Pods :
+```shell
+kubectl get Pods
+```
+
+La sortie va donner :
+```shell
+NAME             READY   STATUS    RESTARTS   AGE
+frontend-pxj4r   1/1     Running   0          5s
+pod1             1/1     Running   0          13s
+pod2             1/1     Running   0          13s
+```
+
+De cette manière, un ReplicaSet peut avoir un ensemble de Pods hétérogène.
+
+## Écrire un manifest de ReplicaSet
+
+Comme avec tous les autres objets API Kubernetes, un ReplicaSet a besoin des champs `apiVersion`, `kind` et `metadata`.
+Pour ReplicaSets, l'attribut `kind` est toujours ReplicaSet.
+
+Dans Kubernetes 1.9, la version de l'API `apps/v1` pour le type ReplicaSet est la version actuelle et activée par défaut. La version de l'API `apps/v1beta2` est obsolète.
+
+Reportez-vous aux premières lignes de l'exemple `frontend.yaml` pour obtenir des conseils.
+
+Un ReplicaSet a également besoin de [`.spec` section](https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#spec-and-status).
+
+### Pod Template
```

Comment:
```sugestion
### Template Pod
```
ou Modèle Pod
?

Reply:
We could put "Modèle de Pod" but since `template` is also an attribute of the ReplicaSet, I'd say the title is more understandable with Pod Template

Reply:
Je suis d'avis de laisser template. Je n'ai jamais entendu ou vu qui que ce soit utilisé modèle dans ce contexte.